spec/unit/views/components/batch_action_selector_spec.rb

```diff
@@ -1,25 +1,21 @@
 require 'rails_helper'
-require 'active_admin/batch_actions/views/batch_action_popover'
+require 'active_admin/batch_actions/views/batch_action_selector'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.